compiler/tests/testSuite.bal

```diff
@@ -69,22 +78,29 @@ function compileErrorLineNumber(CompileError err) returns int? {
     }
 }
 
-function listSourcesVPO() returns TestSuiteCases|error => listSources("vpo");
+function listSourcesVPO() returns TestSuiteCases|error => listSources("vpo", "./testSuite");
 
-function listSourcesEU() returns TestSuiteCases|error => listSources("eu");
+function listSourcesEU() returns TestSuiteCases|error => listSources("eu", "./testSuite");
 
-function listSources(string initialChars) returns TestSuiteCases|io:Error|file:Error {
+function listSources(string initialChars, string path) returns TestSuiteCases|io:Error|file:Error {
     TestSuiteCases cases = {};
     // JBUG #32615 can't use from-in-from query syntax
-    foreach var dir in check file:readDir("./testSuite") {
+    foreach var dir in check file:readDir(path) {
         if !check file:test(dir.absPath, file:IS_DIR) {
             continue;
         }
         string category = check file:basename(dir.absPath);
         foreach var test in check file:readDir(dir.absPath) {
             string name = check file:basename(test.absPath);
             var [base, ext] = basenameExtension(name);
-            if ext != ".bal" {
+            if check file:test(test.absPath, file:IS_DIR) && ext == ".modules" {
```

Comment:
You can just look at test.dir instead of calling file:text

But I don't understand why you are looking at .modules in this function

Reply:
What I am trying to do here is to see if the "test" is actually the modules directory for a test and if so call `listSources` recursively on that directory as well.

Reply:
Why?

Reply:
I thought we need to test we need to test both the imported files separately and then test them after importing. Now removed